js/docEnums.js

```diff
@@ -627,7 +627,7 @@
 
 /**
  * @typedef {string} Enums.DrawerPosition
- * @enum {'left'|'right'|'top'|'bottom'}
+ * @enum {'left'|'right'|'top'|'bottom'|'auto'}
```

Comment:
I think we should have a couple of new values. 
One of it setts Drawer's panel default position by the left side for a LTR locale, and by the right side for a RTL locale(`before`? `start`?).
Another setts Drawer's panel position by the right side for a LTR locale, and by the left side for a RTL locale (`after`? `end`?).

Reply:
Take a look to the experimental [text-align](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align#Values) values.

Reply:
How popular are RTL scenarios? If they are not so popular, in most cases, users can be confused by duplicated values ("left"|"before" & "right"|"after"). And the case when the menu is displayed at the right for LTR and left for RTL looks rare. However, if someone needs to configure Drawer this way, he or she can set this option to "left" or "right" depending on whether the LTR/RTL mode.

Reply:
>How popular are RTL scenarios?
We have feedback related to the Drawer usage with RTL right now

>users can be confused by duplicated values
`left` or `right` are **static** positions. `before` and `after` - adaptive. Use adaptive values for a common scenarios, or use static value to set Drawer's panel position for the any locale.

>However, if someone needs to configure Drawer this way, he or she can set this option to "left" or "right" depending on the LTR/RTL mode.
So, you suggest to drop RTL support for this widget?

Reply:
I suggest to support RTL by default. If position is "auto", the menu is displayed at the left for LTR and at the right for RTL. However, if users ask for adaptive values, "before" and "after" look good.